README.md

```diff
@@ -11,6 +11,9 @@ Open Source Programs (OSP) is an application that simplifies the processing and
 ## API Documentation
 - [Postman Docs](https://documenter.getpostman.com/view/11324046/Szzoaw1q?version=latest)
 
+## Documentation
```

Comment:
there is already a section for documentation a few lines below on README, could you add this sentence there, please?

Reply:
hey @isabelcosta should I remove the line stating about the wiki of the backend & frontend (For more information, you can read backend project wiki and the web project wiki) & add this line or add it separately?